.github/workflows/pull-request-build.yml

```diff
@@ -1,37 +1,42 @@
-name: "Android CI PR Build"
-
-on:
-  pull_request:
-    branches:
-      - "**"
-
-jobs:
-  release:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-      - name: Set env
-        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
-      - name: Set up JDK 12
-        uses: actions/setup-java@v3
-        with:
-          java-version: '12'
-          distribution: 'zulu'
-      - uses: subosito/flutter-action@v2
-        with:
-          channel: 'stable'
-      - name: Set up Flutter
-        run: flutter pub get
-      - name: Generate files with Builder
-        run: flutter packages pub run build_runner build --delete-conflicting-outputs
-      - name: Build with Flutter
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: flutter build apk
-      - name: Add version to APK
-        run: mv build/app/outputs/flutter-apk/app-release.apk revanced-manager-${{ env.RELEASE_VERSION }}.apk
-      - name: Upload APK
-        uses: actions/upload-artifact@v3
-        with:
-          name: revanced-manager-${{ env.RELEASE_VERSION }}
-          path: revanced-manager-${{ env.RELEASE_VERSION }}.apk
+name: PR Build

```

Comment:
I don't think it should build on every PR. At max make it manually dispatchable. Building takes a long time.

Reply:
Alright, will do.